resources/assets/js/app/services/VehicleService.js

```diff
@@ -137,10 +138,35 @@ export const VehicleValidate = (data = {
     return result;
 };
 
+export const VehicleData = {
+    fetchBrand(name) {
+        return simpleRequest.get(`/api/v1/car-brand/`, {
+            params: {
+                search: name,
+                orderBy: 'name'
+            }
+        });
+    },
+
+    fetchModel(name, brandId) {
```

Comment:
if you name method fetch, than it should fetch data with any params, method name should reflect it's logic